grid2op/utils/l2rpn_idf_2023_scores.py

```diff
@@ -166,23 +172,24 @@ def _compute_episode_score(
         nres_score = self.scale_nres_score * nres_score 
         
         #assistant_confidence_score
-        # new_renewable_sources_usage_score_nm = "assistant_confidence_scores"
-        # real_nm = EpisodeStatistics._nm_score_from_attr_name(new_renewable_sources_usage_score_nm)
-        # key_score_file = f"{EpisodeStatistics.KEY_SCORE}_{real_nm}"
-        # assistant_confidence_score = float(other_rewards[-1][key_score_file])
-        assistant_confidence_score = 0 #self.scale_assistant_score * assistant_confidence_score
+        assistant_confidence_score_nm = "assistant_confidence_scores"
+        real_nm = EpisodeStatistics._nm_score_from_attr_name(assistant_confidence_score_nm)
+        key_score_file = f"{EpisodeStatistics.KEY_SCORE}_{real_nm}"
+        assistant_confidence_score = float(other_rewards[-1][key_score_file])
+        assistant_score = self.scale_assistant_score * assistant_confidence_score
         
         #assistant_cost_score
-        # new_renewable_sources_usage_score_nm = "assistant_cost_scores"
-        # real_nm = EpisodeStatistics._nm_score_from_attr_name(new_renewable_sources_usage_score_nm)
+        # assistant_cost_score_nm = "assistant_cost_scores"
+        # real_nm = EpisodeStatistics._nm_score_from_attr_name(assistant_cost_score_nm)
         # key_score_file = f"{EpisodeStatistics.KEY_SCORE}_{real_nm}"
         # assistant_cost_score = float(other_rewards[-1][key_score_file])
-        assistant_cost_score  = 0 #self.scale_assistant_score * assistant_cost_score
+        # assistant_cost_score = max(assistant_cost_score, self.min_assistant_cost_score / self.scale_assistant_score)
+        # assistant_cost_score  = self.scale_assistant_score * assistant_cost_score
         
-        assistant_score = self.weight_confidence_assistant_score * assistant_confidence_score +\
-             (1. - self.weight_confidence_assistant_score) * assistant_cost_score
+        # assistant_score = self.weight_confidence_assistant_score * assistant_confidence_score +\
+        #      (1. - self.weight_confidence_assistant_score) * assistant_cost_score
 
         ep_score = (
             self.weight_op_score * op_score + self.weight_nres_score * nres_score + self.weight_assistant_score * assistant_score
         )
-        return (ep_score, op_score, nres_score, assistant_confidence_score, assistant_cost_score), n_played, total_ts
+        return (ep_score, op_score, nres_score, assistant_score), n_played, total_ts
```

Comment:
just a question: can we add here some extra information like: number of alert sent, true positive, true negative, false negative, false positive ?
(everything should be already computed)

Reply:
how ? do I output more items in the reward or is there an access to other reward attributes htrough EpisodStatistics ?